edx_repo_tools/dev/get_org_repo_names.py

```diff
@@ -0,0 +1,34 @@
+"""Clone an entire GitHub organization into current directory."""
+
+import os.path
+
+import click
+from git.repo.base import Repo
+
+from edx_repo_tools.auth import pass_github
+
+
+@click.command()
+@click.option(
+    '--forks/--no-forks', is_flag=True, default=False,
+    help="Should forks be included?"
+)
+@click.option(
+    '--depth', type=int, default=0,
+    help="Depth argument for git clone",
+)
+@click.argument(
+    'org'
+)
+@click.option(
+    '--output_file', default="repositories.txt")
+@pass_github
+def main(hub, forks, depth, org, output_file):
```

Comment:
clone_org.py didn't have one, but it should, and this should have one too: A docstring, which will get printed as part of the --help output.